client/components/LoginForm/LoginFormDumb.js

```diff
@@ -30,8 +32,27 @@ class LoginForm extends Component {
     }));
   };
 
-  handleSubmit = e => {
+  handleSubmit = async e => {
     e.preventDefault();
+
+    try {
+      const { data } = await Axios.put('/auth/local/login', this.state.values, {
+        validateStatus: function(status) {
+          return status === 200 || status === 401;
+        },
+      });
+
```

Comment:
What is managing our logged in data. Is it our redux store? Or is it the state of our props?
I feel a little conflicted, or maybe a little confused here. 
I'm pretty sure this works fine, I haven't played with it yet. But should a redux dispatch do this?

Reply:
I know its really hard to manage things when they only have one point of state. Like this case.
So I'm not hard headed in the "do it this way" mentality. I'm not even into the purity of things. Because actually I don't really care when it comes to stuff like that. 

I guess I was just a little surprised to see the put request here. Thats all

Reply:
The authentication slice of the redux store manages a logged in user. The function on line 51 dispatches an action with a user. I dislike using thunks with forms. I think they typically mean storing form values in redux, if you don't want to lose them if there was an error from the api, and I dislike storing form values in state.